content/02.introduction.md

```diff
@@ -44,82 +57,70 @@ The current pandemic of COVID-19, caused by the severe acute respiratory syndrom
+Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
+This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].
 
-Replication initiates with the binding of the virus to a host cell via the S protein.
-The viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm.
-The replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
-The dsRNA genome is then transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].
+Most coronaviruses are considered zoonotic viruses with little to no transmission observed in humans.
+A major group of coronaviruses include human coronaviruses (HCoVs) strains associated with multiple respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
+Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
+In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9].
 
-#### Transmission
+The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
+The possibility of asymptomatic cases is also being investigated.
+The transmission and mortality rate estimations of COVID-19 remain to be determined.
```

Comment:
Just curious, I'm not sure what data is publically available, but is this because of the inconsistency in testing? Not sure if we want to mention the reason here...

Reply:
I didn't write this section and agree we may be able to update!

Reply:
I think we should definitely update that and provide a number. As it says it is "only" an estimation and hence only attempts to figure out the rates.

Reply:
I think we may be able to cut this line based on @alavendelm and @rdvelazquez section in Pathogenesis -- curious if they would agree?

Reply:
The [reproduction number section](https://greenelab.github.io/covid19-review/#reproduction-number-and-dynamics-of-transmission) touches on transmission rate but doesn't address mortality rate. That section states, "Estimates of R0 for COVID-19 lie in the range R0=1.4-6.5".

I would be ok with just removing this line or stating the estimated range of R0 and referencing the Reproduction number section.

Reply:
Yes agree. Good estimates are now available for transmission and mortality (tho we don't cover mortality in R0 section). Either remove this sentence, or we can add more details.